Test/GetPowerShellModule.Tests.ps1

```diff
@@ -174,7 +246,46 @@ Describe 'GetPowerShellModule.when called with clean mode' {
         Init
         GivenModule 'Rivet'
         GivenCleanMode
-        WhenPowershellModuleIsRan
-        ThenModuleShouldNotExist
+        WhenTaskRun
+        ThenModuleNotInstalled
     }
 }
+
+Describe 'GetPowerShellModule.when allowing prerelease versions' {
+    AfterEach { Reset }
+    It 'should install a prelease version' {
+        Init
+        GivenModule 'Whiskey'
+        GivenVersion '0.43.*-*'
+        GivenPrereleaseAllowed
+        WhenTaskRun
+        ThenModuleInstalled -AtVersion '0.43.0-beta1416'
+    }
+}
+
+Describe 'GetPowerShellModule.when installing to custom directory' {

```

Comment:
Add a test that ensures we can install modules at both the default and custom locations during a build?

Reply:
The test after this one handles that case.